### ISSUE TITLE
Multi label object detection

### ISSUE THREAD

**Issue by HenryPeking** [open-mmlab/mmdetection#10828]
**Enable multi label object detection**

**Motivation**
Multilabel object detection is an important task in computer vision, as it allows for the detection and recognition of multiple objects within an image. This is particularly useful in applications such as surveillance, where it is necessary to identify multiple objects and their relationships within a scene. Multilabel object detection can also be used in tasks such as image retrieval and scene understanding, where the ability to accurately identify multiple objects can improve the performance of these systems. Overall, multilabel object detection plays a crucial role in enabling machines to understand and interpret visual information.

**Comment by hhaAndroid** [comment]
@HenryPeking Can you provide a few representative algorithms or mainstream algorithm names? We are going to conduct some research.

**Comment by HenryPeking** [comment]
I recognize ICLR2023 accepted paper -- [Medical Image Understanding with Pretrained Vision Language Models: A Comprehensive Study.](https://arxiv.org/abs/2209.15517v2) which based on GLIP and may be a possible solution. It is also a OpenMMLab's work. However the maskrcnn_benchmark utilized in the code is hard to compile in my platform and the finetune code has not been released, yet. Therefore, I am wondering if it is possible for MMDetection to adapt it?

**Comment by hhaAndroid** [comment]
@HenryPeking Currently, GLIP supports inference, and fine-tuning will be supported in the future.

**Comment by HenryPeking** [comment]
@hhaAndroid May I ask when will the finetune code release?
